src/hotspot/share/runtime/javaThread.hpp

```diff
@@ -714,6 +713,9 @@ class JavaThread: public Thread {
   MemRegion deferred_card_mark() const           { return _deferred_card_mark; }
   void set_deferred_card_mark(MemRegion mr)      { _deferred_card_mark = mr;   }
 
+  bool in_internal_oome_mark() const            { return _in_internal_oome_mark; }
+  void set_in_internal_oome_mark(bool b)        { _in_internal_oome_mark = b;    }
+
```

Comment:
Should all these be prefixed with `is` like:
```
  bool is_in_VTMS_transition() const             { return _is_in_VTMS_transition; }
  bool is_in_tmp_VTMS_transition() const         { return _is_in_tmp_VTMS_transition; }
  bool is_in_any_VTMS_transition() const { return _is_in_VTMS_transition || _is_in_tmp_VTMS_transition; }
```

Reply:
Ok.